src/styles/globals.css

```
@@ -5,12 +5,15 @@
    THEME — font & screen tokens for Tailwind
 ═══════════════════════════════════════ */
 @theme {
-  --font-sans: var(--font-epilogue), system-ui, sans-serif;
-  --font-serif: var(--font-fraunces), Georgia, serif;
-  --font-mono: 'JetBrains Mono', 'Fira Code', monospace;
+  /* The three faces come from the active pairing in src/utils/fonts/index.ts,
+     which declares them on <html> as --font-*-face. Nothing here names a
+     typeface directly, so switching pairings is a one-line change there. */
+  --font-sans: var(--font-text-face), system-ui, sans-serif;
+  --font-serif: var(--font-display-face), Georgia, serif;
```

Stylelint (17.14.1)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)